integration_test/third_party_apps_data/applications/aerospike/metadata.yaml

```diff
@@ -21,7 +21,13 @@ description: |-
   and connections. The integration collects these metrics using the official
   client API provided by Aerospike.
 supported_app_version: ["4.9", "5.x", "6.x"]
-configure_integration: ""
+configure_integration: |-
+  Aerospike logs are not supported on systems such as SLES where Aerospike
```

Comment:
```suggestion
  Aerospike logs are not automatically ingested on systems such as SLES where Aerospike
```

Reply:
Agree w/ Igor

Reply:
On second thought, maybe it's better to talk about what _does_ happen, instead of what does not. E.g.:
```
  Aerospike logs are automatically ingested on systems where Aerospike runs as 
  a `systemd` service. On other systems such as SLES, you can manually…
```

Reply:
I like that too -- updated.